pkg/adapter/adapter.go

```diff
@@ -112,7 +111,7 @@ func (a *Adapter) RequestPageFromDatasource(
 			}...,
 		),
 
-		// SCAFFOLDING:
+		// SCAFFOLDING #25 - pkg/adapter/adapter.go: Uncomment to set the default timezone in case the ingested datetime attribute does not have timezone specified
```

Comment:
To be consistent with SoR terminology used in other comments.
```suggestion
		// SCAFFOLDING #25 - pkg/adapter/adapter.go: Uncomment to set the default timezone in case the SoR datetime attribute does not have timezone specified
```